Johannes storage

JEM-Mosig opened the pull request:
* Enable tracking of tab-switching with new SelectTaskCommand
* Implement semi-automatic performance tracking / dialogue evaluation

Commit:
Store linear user guide instructions

Commit:
Update review_work

Commit:
Implement worker evaluation script

Commit:
Introduce SelectTopicCommand

Commit:
Merge from suggestions-2

Commit:
Fix scenario connection

Commit:
Update score function

Commit:
Merge branch 'woz' into johannes-storage

tttthomasssss reviewed (commented):
Generally LGTM, just a question whether the print statement in the suggestions module was intentionally left in?

Commit:
Implement evaluation script

Commit:
Remove print from suggestions

Commit:
Merge branch 'woz' into johannes-storage

# Conflicts:
#	parlai/mturk/tasks/woz/scenarios/test_mix.json

Commit:
Fix trailing comma in json

JEM-Mosig commented:
> Generally LGTM, just a question whether the print statement in the suggestions module was intentionally left in?

Right, I removed the print. I also merged `woz` and noticed that one of the intent2reply jsons was broken (additional comma in the end is only allowed in python, not in json). Fixed this.

Commit:
Fix trailing comma in json

Commit:
Improve world data storage

Commit:
Merge branch 'suggestions-2' into johannes-storage

# Conflicts:
#	resources/hotel_reserve/intent2reply.json
#	resources/party_plan/intent2reply.json
#	resources/party_rsvp/intent2reply.json
#	resources/plane_reserve/intent2reply.json
#	resources/plane_search/intent2reply.json
#	resources/restaurant_reserve/intent2reply.json
#	resources/restaurant_search/intent2reply.json

Commit:
Fix handling unknown scenarios

Commit:
Stop tracking scenarios

Commit:
Store domain list separately

Commit:
Write store-dialogue script

Commit:
Wrap text in suggestion buttons

Commit:
Don't safe form description

Commit:
Fix 'my hotel' issue

Commit:
Re-introduce onboarding tutorial for now

Commit:
Fix broken replies

Commit:
Weaken rejection criteria for now

Commit:
Merge branch 'woz' into johannes-storage
